posthog/api/test/__snapshots__/test_feature_flag.ambr

```diff
@@ -445,8 +445,8 @@
   SELECT "posthog_person"."uuid"
   FROM "posthog_person"
   WHERE ("posthog_person"."team_id" = 99999
-         AND "posthog_person"."uuid" IN ('00000000000040008000000000000000'::uuid,
```

Comment:
I had to add rules to blank UUIDs in snapshots because the file system is stored as a UUID, and can be referenced directly (`AND NOT ("posthog_filesystem"."id" = '018c8b9a62807ab0823db83a2b8d01ec'::uuid))`)